aws-params-secrets-setup/variables.tf

```diff
@@ -1 +1,5 @@
-
+variable "alias_name" {
+  description = "Chamber is hard coded to use a KMS alias with the name 'parameter_store_alias'."
```

Comment:
should that be `parameter_store_key` instead of `parameter_store_alias`

Reply:
Yeah